src/Cameras/VR/babylon.vrExperienceHelper.ts

```diff
@@ -256,6 +256,8 @@ module BABYLON {
         private _webVRrequesting = false;
         // Are we presenting to the headset right now?
         private _webVRpresenting = false;
+        // Have we entered VR?
+        private hasEnteredVR: boolean;

```

Comment:
_hasEnteredVR to be consistent naming with other private variables. Also, if you could add more of a description to this variable and _webVRpresenting, it would be good. I think _webVRpresenting is what the vrDevice says and _hasEnteredVR is the state of the vrExperienceHelper.

Reply:
I think this goes back to this discussion (https://github.com/BabylonJS/Babylon.js/pull/4909#discussion_r209035219).